Formula/act.rb

```diff
@@ -0,0 +1,28 @@
+class Act < Formula
+  desc "Run your GitHub Actions locally 🚀"
+  homepage "https://github.com/nektos/act"
+  url "https://github.com/nektos/act/archive/v0.2.10.tar.gz"
+  sha256 "1e4d371cfa6aec7e350927e55c3ac8413dbf6e23edbb811ebe2d587dbf136928"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build", "VERSION=#{version}"
+    bin.install "dist/local/act"
```

Comment:
Once https://github.com/nektos/act/pull/308 lands in this can be simplified to:
```rb
  system "make", "install", "PREFIX=#{prefix}", "VERSION=#{version}"
```

Reply:
Or just:
```ruby
system "go", "build", *std_go_args, "-ldflags", "-X main.version=#{version)"
```

Reply:
@dawidd6 Sure, that is also a valid option but I tend to favor makefiles from the future-proof standpoint. If we replicate the current state of Makefile we would need to constantly keep it in sync with possible future changes. OTOH using Makefile requires less manual work in the long run.